code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine_vr.dm

```diff
@@ -76,3 +77,18 @@
 		chem_effective = 0.75
 	if(alien != IS_DIONA)
 		M.heal_organ_damage(8 * removed * chem_effective, 0)
+
+/datum/reagent/sleevingcure
```

Comment:
I'd suggest maybe tossing a teeny handful of these into the vault for emergencies. It would be neat fluff to have them there as well.

Reply:
I'll throw in a half-full pill bottle into the Vault then

Reply:
My intention of having them tossed into ERT is that a bad enough emergency requiring people to get treated for resleeving sickness will probably also prompt an ERT call, since ERT themselves would not normally require the meds, they can bring the pill bottle to help treat Tether crew.

Reply:
...might instead toss only a single pill into the Vault, moreso for posterity and "final solution" type of thing.

Reply:
Single pill in the vault is fine, and give ERT a full pill bottle (14 pills) instead of the usual half-full (7 pills) that most prespawned pill bottles get.

Reply:
A package with a single KHI Cure pill added to Vault freezer.